redwood-layout/src/commonMain/kotlin/app/cash/redwood/layout/properties.kt

```diff
@@ -25,10 +25,7 @@ import kotlinx.serialization.Serializable
 /** Equivalent to `justify-content`. */
 @JvmInline
 @Serializable
-public value class MainAxisAlignment(public val ordinal: Int) {
-  init {
-    require(ordinal in 0..5) { "Unknown MainAxisAlignment: $ordinal" }
-  }
+public value class MainAxisAlignment private constructor(public val ordinal: Int) {
```

Comment:
Making these constructors private otherwise autocomplete always autocompletes the constructor instead of the constants.